### DISCUSSION TITLE
docs/istio-handbook/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/cloud-native-library#5] [Announcements]
# docs/istio-handbook/

云原生应用网络构建指南。

https://jimmysong.io/docs/istio-handbook/

**Comment by rootsongjc** [comment]
这本书是从原来的 [GitBook](https://jimmysong.io/istio-handbook/) 迁移而来，并做了以下优化：
- 使用 Hugo 主题重新构建，页面更加现代化，比起 Gitbook V1 早就过时且不再更新了
- 修改了文中的链接错误，以后只维护这个页面上的内容了，老的 Gitbook 内容就不再维护了
- 增加了评论功能（使用的是 Giscus）
